types/expression-evaluator/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for expression-evaluator 0.2
+// Project: https://github.com/GreetzNL/expression-evaluator#readme
```

Comment:
Please provide the link to repo that works. This link is not accessible.

Reply:
Thanks for feedback. The link to repo is working within our organisation. It doenen't work for you because of you are not a contributor of GreetzNL. Is it ok to keep this like that?

Reply:
If that's the case why not host the definitions inside the package itself.
Without working project link, we cannot verify the definitions represent correct structure or not.

Reply:
We're doing similar thing now. We are using deprecated Typings package and keep all .d.ts fils there. But now we decided to upgrade to @types.
We will discuss with our team and organization what we can do about the link and I will update you.
Thanks